db/mongo_test.go

```diff
@@ -0,0 +1,76 @@
+package db
+
+import (
+	"os"
+	"testing"
+)
+
+func TestMongoDB(t *testing.T) {
+	id := "33683111000280"
+	json := `{"qsa": [{"name": 42}, {"name": "forty-two"}], "answer": 42}`
+
+	u := os.Getenv("MONGO_URL")
+	if u == "" {
+		t.Errorf("expected a posgres uri at TEST_DATABASE_URL, found nothing")
+		return
+	}
+	mdb, err := NewMongoDB("minhareceita")
```

Comment:
Não temos outra `db` nesse escopo, né?

```suggestion
	db, err := NewMongoDB("minhareceita")
```

Reply:
feito